.circleci/config.yml

```diff
@@ -264,6 +264,23 @@ jobs:
       - simulate_nested:
           task: "/sep/zora-002-fp-upgrade"
 
+  simulate_sequence:
+    description: "Runs a sequence of simulations"
+    parameters:
+      prefix:
+        type: string
+      tasks:
+        type: string
+      block_number:
+        type: string
```

Comment:
is `latest` supported, so we can continually run it against the latest block for in-progress tasks?

Reply:
since `block_number` is optional, not setting it would be resulting to simulate against the latest block.

Reply:
Ok perfect, thank you! So for tests we should specify a block number like you did, and in practice when we go to use this we will want to use latest